types/models/DropdownData.ts

```diff
@@ -0,0 +1,7 @@
+export interface DropdownItem {
+    label: string;
+    value: {
```

Comment:
Rather than having `value: { onPress: () => void }` lets separate these. That makes the dropdown available for radio type selects.